### PR TITLE
[MISC] face landmarks detection

### PR DESCRIPTION
**Pull request by ygnn123**

### CODE REVIEW

**Review by druzhkov-paul — CHANGES_REQUESTED** [review]
General comments:

1. Please remove irrelevant code.
2. If you want to reuse some functionality, that was not implemented by you, please do that explicitly: base off of some branch, not just copy-and-paste code.
3. Please add readme file, where provided models are listed and steps to run training/evaluation/export are described. Take a look [here](https://github.com/openvinotoolkit/training_extensions/tree/develop/models/object_detection/model_templates/face-detection) for reference.

File: ote/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-mmcv
+mmcv==1.2.7
```
[the remark this answers is not in the document]
Comment (ygnn123) [reply ote/requirements.txt:1]: Pip installs 1.3.5 version by default.  And last mmpose version have assert on version <1.3.0. Ok, I'm going change it. Thanks.

**Comment by Ilya-Krylov** [comment]
@ygnn123 close or update

**Comment by nervana-ff** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**